### PR TITLE
add a script to pull interesting stats from a zork log

### PR DESCRIPTION
**Pull request by trevj**
This is really bad and slow but I've found it useful to assemble a few basic stats for our beta testers. Think of it as a placeholder for now.

### CODE REVIEW

File: testing/run-scripts/zork-stats.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    esac
+done
+shift $((OPTIND-1))
+
+if [ $# -lt 1 ]
+then
+    usage
+fi
+
+IFS="
```
Comment (soycode) [inline testing/run-scripts/zork-stats.sh]: Might be worth commenting this as "set field separator to only newlines", as I had to look it up at least.

Comment (trevj) [inline testing/run-scripts/zork-stats.sh]: Done! Agreed, it's a...lesser-known feature :-)

**Comment by soycode** [comment]
:+1: - seems straightforward, if it's slow then grep must be expensive in this case. Maybe some mix of awk/sed wizardry could speed things up, but agreed that this is fine to check in for now.

**Comment by trevj** [comment]
Right! I could think of a bunch of ways to improve this but then the script is getting a little complicated. Thanks!

**Pull request merged**